extensions/amp-instagram/0.1/amp-instagram.js

```diff
@@ -61,7 +61,10 @@ class AmpInstagram extends AMP.BaseElement {
      * @private {?Promise}
      */
     this.iframePromise_ = null;
-
+    /**
+     * @private {?Element}
+     */
+    this.placeholderWrapper_ = null;
```

Comment:
Yeah, I thought about that but the problem with that is it would be hidden by the returned imagePromise inside `layoutCallback`, no?